sql-parser/src/main/java/io/crate/sql/tree/CreateTable.java

```diff
@@ -109,6 +109,12 @@ public PrivilegeType privilegeType() {
     }
 
     @Override
-    public String clazz() { return "SCHEMA";}
+    public PrivilegeClazz privilegeClazz() {
+        return PrivilegeClazz.SCHEMA;
+    }
 
+    @Override
+    public String privilegeIdent() {
+        return name.getName().getParts().get(0);
```

Comment:
this would result in the table name if no schema is used at the statement. only return the first part if size is > 1, otherwise null (we must then use the sessions default schema as an ident)